modules/vault-secrets.nix

```diff
@@ -11,29 +11,7 @@ let
     {name, ... }:
     {
       options = {
-        namespace = mkOption {
-          type = with types; str;
-          default = cfg.namespace;
-          example = "servers/jupiter/services";
-          description = ''
-            Vault KV path under which all service secrets live, under
-            <literal>basePath</literal>. No leading or trailing slash!
-          '';
-        };
-
-        environmentKey = mkOption {
-          type = with types; nullOr str;
-          default = "environment";
-          example = "environment_data";
-          description = ''
-            Vault KV key under <literal>vaultPathPrefix/namespace</literal> that
-            contains the environment for this service. Keys will be dumped into
-            <literal>outPathPrefix/name/environment</literal> in a format
-            suitable for use with EnvironmentFile.
-          '';
-        };
-
```

Comment:
Hm, what would we use instead?

Reply:
Rather than `vaultPrefix/namespace`, now there's only `vaultPrefix`.

Reply:
Ok, I guess it makes sense